example/spec/parallel/ingest/ingestFromPdrSpec.js

```diff
@@ -159,9 +150,11 @@ describe('Ingesting from PDR', () => {
       workflowExecution = await buildAndExecuteWorkflow(
         config.stackName,
         config.bucket,
-        taskName,
-        collection,
-        provider
+        workflowName,
+        undefined,
```

Comment:
So this doesn't workflow really doesn't need a collection because it's a PDR based workflow?

Reply:
Apparently not. The `discoverPdrs` task does not use collection, and the `queuePdrs` task takes a collection, but it's optional and is only passed along to the triggered workflow. The workflow triggered by `queuePdrs` is `parse_pdr_workflow`, which does not use a collection.